components/color-palette-custom/color-palette-custom.js

```diff
@@ -1,5 +1,5 @@
+import React, { useState } from 'react'; // eslint-disable-line no-unused-vars
```

Comment:
So this is because of Storybook? Because you're already importing react in `story.js` here: https://github.com/infinum/eightshift-frontend-libs/blob/1558e1e364f7688a99fb9db90c401fe97cfc2baf/components/color-palette-custom/docs/story.js

That isn't enough?

Reply:
Yes, you have to import it in to every single file that is used inside of storybook